Gulpfile.js

```diff
@@ -1,33 +1,42 @@
-var autoprefix = require("gulp-autoprefixer"),
-    connect    = require("gulp-connect"),
-    gulp       = require("gulp"),
-    sass       = require("gulp-sass");
+const browserSync = require('browser-sync').create();
+const gulp = require('gulp');
+const sass = require('gulp-sass');
 
-var paths = {
-  scss: [
-    "./core/**/*.scss",
-    "./contrib/**/*.scss"]
+const paths = {
+  markup: {
+    src: './contrib/**/*.html',
+  },
+  styles: {
+    src: [
+      './contrib/**/*.scss',
+      './core/**/*.scss',
+    ],
+    dest: './contrib/',
+  }
 };
 
-gulp.task("sass", function () {
-  return gulp.src(paths.scss)
-    .pipe(sass({
-        sourcemaps: true,
-        precision: 6
-    }))
-    .pipe(autoprefix("last 2 versions"))
-    .pipe(gulp.dest("./contrib"))
-    .pipe(connect.reload());
-});
+function styles() {
```

Comment:
Why are we changing the syntax of all of these tasks? This isn't typically the way one would author a guplfile.

Reply:
This is a feature of gulp v4 and appears to be the recommended way. I followed the gulp docs recipe to get to this point: https://github.com/gulpjs/gulp/blob/master/docs/recipes/minimal-browsersync-setup-with-gulp4.md